PyPortal_ObliqueStrategies/oblique.py

```diff
@@ -0,0 +1,44 @@
+"""
+This code will display a random strategy from strategies.py when the
+PyPortal screen is pressed. See the original Oblique Strategies
+by Brian Eno & Peter Schmidt here: https://www.enoshop.co.uk/product/oblique-strategies
+"""
+import os
+import random
+import board
+from strategies import strategies
+from adafruit_pyportal import PyPortal
+
+cwd = os.getcwd() # the current working directory (where this file is)
+wrap = 35 # text wrap value
+
+# create pyportal object w no data source (we'll feed it text later)
+pyportal = PyPortal(url = None,
+                    json_path = None,
+                    status_neopixel = board.NEOPIXEL,
+                    default_bg = None,
+                    text_font = cwd+"fonts/Arial-ItalicMT-17.bdf",
+                    text_position = (30, 100),
```

Comment:
why not center the text on (30, 120)? is this on purpose?

Reply:
Ya - 100 was from when I tested with a larger font. I'll set it to  30, 120 in the next commit.